cdk/nbconvert_stack.py

```diff
@@ -56,6 +56,11 @@ def setup_api_gateway(
         cert_arn: str,
         base_path: str) -> apigw2.HttpApi:
 
+        log_group = logs.LogGroup(
+            self, "HttpApiAccessLogs",
```

Comment:
Is "HttpApiAccessLogs" the name of the log group and, if so, is it specific enough for the stack?  E.g., should it be something like "NBConvertApiAccessLogs", to differentiate from access logs for another stack?

Reply:
Good point, renamed to "NBConvertApiAccessLogs".